docs/20_smart-contracts/40_guides/10_resources_guide.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Resources Guide
+---
+
+The EOS blockchain works with three system resources: CPU, NET and RAM. The EOS accounts need sufficient system resources to interact with the smart contracts deployed on the blockchain.
+
+* [RAM Resource](#ram-resource)
+* [CPU Resource](#cpu-resource)
+* [NET Resource](#net-resource)
+
+To allocate resources to an account use the `PowerUp Model` to [power up that account](#account-power-up).
+
+## RAM Resource
+
+RAM is the memory, the storage space, where the blockchain stores data. If your contract needs to store data on the blockchain, like in a database, then it can store it in the blockchain's RAM in the form of a `multi-index table` or a `singleton`.
+
+### RAM High Performance
+
+The EOS blockchain high performance is achieved also because the data stored on the blockchain uses RAM as the storage medium, and the access to blockchain data is very fast. The EOS blockchain performance benchmarks reach levels very few blockchains have been able to, and it elevates EOS blockchain one of the few contenders for the fastest blockchain of the world.
+
+### RAM Importance
+
+RAM is a very important system resource because of the following reasons:
+
+* RAM is a limited resource, the public EOS blockchain started with 64GB of RAM and after that the block producers decided to increase the memory with 1KB per block. This way the supply of RAM constantly increases and its price does not grow too fast too high due to the increased demand from blockchain applications.
```

Comment:
Let's leave historical contexts out of docs. You can just say "RAM is a limited resource, with only 64GB available for everyone. This can be expanded later by the block producers."